<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1116</width>
    <height>781</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cinema Stats V1.0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="MainTabMenu">
    <property name="geometry">
     <rect>
      <x>-4</x>
      <y>39</y>
      <width>1121</width>
      <height>741</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="HomeTab">
     <attribute name="title">
      <string>Home</string>
     </attribute>
     <widget class="QLabel" name="LastTenLabel">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Last 10 Films</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="LastTenTable">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>831</width>
        <height>561</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="columnCount">
       <number>0</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="LogViewTab">
     <attribute name="title">
      <string>Log View</string>
     </attribute>
     <widget class="QTableView" name="MainLogTable">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>831</width>
        <height>561</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="LoggedFilmsLabel">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Logged Films</string>
      </property>
     </widget>
     <widget class="QComboBox" name="SortByDropdown">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>30</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Date Logged</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Movie Title (Ascending)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Movie Title (Descending)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Rating</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Genre</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="SortByLabel">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>20</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Sort By:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="InsightsTab">
     <attribute name="title">
      <string>Insights</string>
     </attribute>
     <widget class="QStackedWidget" name="InsightsStackedWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>1061</width>
        <height>681</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <widget class="QWidget" name="MoviesOverTimePlaceholder" native="true">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>10</y>
          <width>911</width>
          <height>621</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>5</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);</string>
        </property>
       </widget>
       <widget class="QLabel" name="MoviesVersusTimeLabel">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>640</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Movies Versus Time</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QWidget" name="GenrePieChartPlaceholder" native="true">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>80</y>
          <width>831</width>
          <height>541</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>5</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(170, 0, 0);</string>
        </property>
       </widget>
       <widget class="QLabel" name="GenrePieChartLabel">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>630</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Genre Pie Chart</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="AddMediaButton">
    <property name="geometry">
     <rect>
      <x>1034</x>
      <y>10</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Media</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SettingsButton">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>10</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Settings</string>
    </property>
   </widget>
   <widget class="QLabel" name="CinemaStatsLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cinema Stats</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
